lib/msal-node/test/client/ManagedIdentitySources/MachineLearning.spec.ts

```diff
@@ -71,6 +76,15 @@ describe("Acquires a token successfully via an Machine Learning Managed Identity
             expect(networkManagedIdentityResult.accessToken).toEqual(
                 DEFAULT_USER_SYSTEM_ASSIGNED_MANAGED_IDENTITY_AUTHENTICATION_RESULT.accessToken
             );
+
+            const url: URLSearchParams = new URLSearchParams(
+                sendGetRequestAsyncSpy.mock.lastCall[0]
+            );
+            expect(
+                url.has(
+                    ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID_2017
+                )
+            ).toBe(true);
```

Comment:
While the test confirms the “happy path” for ML 2017, it doesn’t confirm that the code avoided incorrectly including “client_id” at the same time. Would it be good to check for,  

```js
expect(
  url.has(
    ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID
  )
).toBe(false);

```

Reply:
you can see how in MSAL .NET we test this. i.e. testing the not expected param value - https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/5fcc057c08fbb8cf4fe7a231558557be398dddaa/tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHttpMessageHandler.cs#L125

Reply:
done